web-app/js/portal/cart/Downloader.js

```diff
@@ -84,6 +88,25 @@ Portal.cart.Downloader = Ext.extend(Object, {
         });
     },
 
+    _onDownloadRequestSuccess: function(response, request) {
+
+        var url = request.scope.proxyUrl;
+
+        if ($downloaderLink && $downloaderLink.length > 0) {
+            $downloaderLink.attr('src', url);
+        } else {
+            $downloaderLink = $('<iframe>', { id:'downloaderLink', src: url }).hide().appendTo('body');
+        }
+    },
+
+    _onDownloadRequestFailure: function(resp) {
+        log.error("Synchronous download request has failed with the following error " + resp.responseText);
+        Ext.Msg.alert(
```

Comment:
I'd prefer to see some sort of error indicator/msg next to the row in the download page, rather than a popup.  This could tie in nicely with work to follow on from https://github.com/aodn/aodn-portal/pull/1684.